### PR TITLE
Don't catch commented includes in the build_for_portal script

### PR DESCRIPTION
**Pull request by aireilly**
Update build_for_portal:
* Ignore commented includes

### CODE REVIEW

**Comment by mramendi** [comment]
Thanks, I'll try to process this week or early next week - I need to do "regression tests" on this

**Comment by aireilly** [comment]
@mramendi don't waste time testing this. It doesn't work

**Comment by aireilly** [comment]
This is very odd. @mramendi I think this update does what it should but it appears to throw up another unrelated but valid error. I think our build is very very fragile :rofl:

File: welcome/index.adoc [hunk 1]
```diff
@@ -9,6 +9,14 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 [.lead]
+// xref:../../this/should.adoc#not-trigger-an-error[test]
+
+//* xref:../../this/should.adoc#not-trigger-an-error-either[test]
+
+////
```
[the remark this answers is not in the document]
Comment (aireilly) [reply welcome/index.adoc:16]: https://github.com/openshift/openshift-docs/pull/83757/files

**Comment by mramendi** [comment]
I did some spot checks on a diff between the output of the current script and this version. All changes look good.

I would ideally like to stage this change to the script to enterprise-4.18 and ensure all builds in pantheon succeed before rolling out to published releases and layered products. However, I am not sure how to implement this with the Prow system.

**Review by mramendi — APPROVED** [review]

**Comment by aireilly** [comment]
This change is so trivial I think we can just merge it to main. @gabriel-rh @gaurav-nelson WDYT

**Pull request merged**